app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -37,28 +39,77 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
         <Form className="w-full">
           <Pane.Body className="space-y-6">
             <Input
+              required
               label="Title"
               name="title"
               className="w-full flex-grow-0"
-              required
             />
             <Textarea
+              required
               label="Description"
               name="description"
+              placeholder="Enter note description"
+              className="w-full flex-grow-0"
+              rows={3}
+            />
+            <Select
+              defaultValue={{
+                label: "Jacob Jones",
+                value: "Jacob Jones",
+              }}
+              isSearchable
```

Comment:
Implicit true props ought to be the first among prop list. [Refer](https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Also%20note%20that%20these%20attributes%20ought%20to%20be%20the%20first%20among%20the%20list%20of%20attributes%20being%20passed%20down%20as%20props%20because%20it%20gives%20a%20better%20visual%20hierarchy%20and%20also%20these%20are%20the%20props%20that%20don%E2%80%99t%20change%20often)

Reply:
Resolved in PR #13